src_py/apiServer/transmitter.py

```diff
@@ -96,15 +100,16 @@ def predict(self):
 
         self.clientsPredict()
 
-        self.updateCSV("Training", globe.trainResults)
+        self.updateCSV("Prediction", globe.predictResults)
 
         while globe.pendingAcks > 0:
             time.sleep(0.005)
             pass 
 
         # 1 Ack for startCasting():
         globe.pendingAcks += 1
-        self.startCasting()
+
+        self.startCasting(10)
```

Comment:
What does it means startCasting(10) ? why 10 and not 15? 
Please use macro like or add a short comment